RecoPixelVertexing/PixelLowPtUtilities/bin/ClusterShapeAnalyzer.cc

```diff
@@ -0,0 +1,402 @@
+#include <cmath>
+
+#include <utility>
+#include <vector>
+#include <map>
+#include <fstream>
+#include <iostream>
+
+using namespace std;
+
+#include "TROOT.h"
+#include "TFile.h"
+#include "TH2F.h"
+#include "TH1F.h"
+#include "TNtuple.h"
+
+//#include "gnuplot_i.hpp"
+//Gnuplot * g;
+
+// Total loss, loss in one direction is Loss/2 
+#define Loss 0.005
+
+// FIXME
+#define MinEntries 100
+
+TH1D * line;
+
+#define exMax 10
+#define eyMax 15
+
+const char *subName[2] = {"barrel","endcap"};
```

Comment:
this and line 26 generates warnings in the static analyzer about
non-const global static variable

@Dr15Jones @smuzaffar  we do not run analyzer on /test directories but apparently do on /bin

In this case, I think the code quality requirements do not need to be strict.
Is there a way to ignore in some central space or should this file be moved to /test, or is there some /testbin?